keyboard_rewrite/src/lib/y2_keyboard/keyboard_layout.rb

```diff
@@ -0,0 +1,26 @@
+require "cheetah"
+
+module Y2Keyboard
+  class KeyboardLayout
+    attr_reader :code
+    attr_reader :description
+
+    LAYOUT_CODE_DESCRIPTIONS = {
+      "es" => "Spanish",
+      "fr" => "French",
+      "us" => "English (US)"
+    }
+
+    def initialize(code, description)
+      @code = code
+      @description = description
+    end
+
+    def self.all
+      raw_layouts = Cheetah.run("localectl", "list-keymaps", stdout: :capture)
```

Comment:
I like the initial API and implementation of this class... to be the initial one. :wink:

Take into account that in a close future we have to make this relatively independent of systemd (likely using some pattern like Strategy). But doing it now would be premature, of course.

Reply:
Yes, I have that in mind, but the API is to small yet, so I prefer to keep it simple for now.